docs/autorenew-feature.md

```diff
@@ -0,0 +1,71 @@
+# The autorenew feature
+
+## Goal
+-	Allow Hedera Services to automatically search for entities such as accounts, files, smart contracts, topics, tokens... that are about to expire and renew them.
```

Comment:
Is there a limitation of entities? If so you could say all entities expect `x, y, z`. 
If not either explicitly name all entities (`Schedules` are missing) or say 'all entity types'

Reply:
There is, @Nana-EC. Schedule transactions never autorenew.

Reply:
Okay, you should be explicit and say that. The current languages makes it seems like you are just giving examples of entities but doesn't imply the limitation.